src/libs/actions/IOU.js

```diff
@@ -724,11 +721,327 @@ function payIOUReport({
     return promiseWithHandlers;
 }
 
+/**
+ * @param {Object} report
+ * @param {Number} amount
+ * @param {String} currency
+ * @param {String} comment
+ * @param {String} paymentMethodType
+ * @param {String} managerEmail - Email of the person sending the money
+ * @param {Object} recipient - The user receiving the money
+ * @returns {Object}
+ */
+function getSendMoneyParams(report, amount, currency, comment, paymentMethodType, managerEmail, recipient) {
+    const newIOUReportDetails = JSON.stringify({
+        amount,
+        currency,
+        requestorEmail: OptionsListUtils.addSMSDomainIfPhoneNumber(recipient.login),
+        comment,
+        idempotencyKey: Str.guid(),
+    });
+
+    const recipientEmail = OptionsListUtils.addSMSDomainIfPhoneNumber(recipient.login);
```

Comment:
if this is used on the newIOUReportDetails it can go before it and used there

Reply:
Done.